### PR TITLE
Leaderboard: setup project 🌱 

### PR DESCRIPTION
**Pull request by ismailmunyentwari9**
# What is done in this project 🥇 

- [ ] Use Gitflow
- [ ] Create an npm project with webpack
- [ ] Write plain HTML markup with minimum styling
- [ ] Use ES6 modules, with import and export.
# Skills Obtained 🌴 

- [ ] Use callbacks and promises.
- [ ] Learn how to use proper ES6 syntax.
- [ ] Use ES6 modules to write modular JavaScript.
- [ ] Use webpack to bundle JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,140 @@
...
+- **[Add-score-form]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://ismailmunyentwari9.github.io/Leaderboard-/dist)
+
```
Comment (Owusu-Desmond) [inline README.md]: - [ ] Please your readme looks awesome :+1:  but your live demo link is broken link so make sure you fix it. :pray: 
  image

File: README.md [hunk 2]
```diff
@@ -1 +1,140 @@
...
+I would like to thank [Microverse](https://www.microverse.org/), for the original [design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE.md) licensed.
+
```
Comment (Owusu-Desmond) [inline README.md:139]: - [ ] Please the link to your license file is a broken link so please make sure that you fix it. :pray: 
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @ismailmunyentwari9 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by huseyinbiyik — APPROVED** [review]
Hi @ismailmunyentwari9,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- 3D button look beautiful 👌 
- Linter set up well and checks pass 🟢
- Githflow followed 👌
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
- Please use descriptive branch names instead of "task-one" (e.g.: refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. [Check](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) more to learn more about this.
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**